src/backup.rs

```diff
@@ -32,6 +51,27 @@ impl MegolmV1BackupKey {
     pub fn backup_algorithm(&self) -> String {
         self.inner.backup_algorithm().into()
     }
+
+    /// Try to create a [`MegolmV1BackupKey`] from a base 64 encoded string.
+    #[napi(strict)]
+    pub fn from_base64(key: String) -> napi::Result<MegolmV1BackupKey> {
+        Ok(Self { inner: InnerMegolmV1BackupKey::from_base64(&key).map_err(into_err)? })
+    }
+
+    #[napi(strict)]
+    pub async fn encrypt(&self, exported_session_json: String) -> napi::Result<KeyBackupData> {
+        let exported_session = serde_json::from_str(&exported_session_json).map_err(into_err)?;
+        let session = InboundGroupSession::from_export(&exported_session).map_err(into_err)?;
+        let res = self.inner.encrypt(session).await;
+        Ok(KeyBackupData {
+            first_message_index: res.first_message_index.into(),
+            forwarded_count: res.forwarded_count.into(),
+            is_verified: res.is_verified,
+            session_data_ephemeral: res.session_data.ephemeral.to_string(),
+            session_data_ciphertext: res.session_data.ciphertext.to_string(),
+            session_data_mac: res.session_data.mac.to_string(),
+        })
```

Comment:
Will this response be used anywhere?  If the job of this method is to encrypt the data, it seems odd to also return that data.  Then again, it's returned by `InnerMegolmV1BackupKey.encrypt`, so it's a pattern we could repeat here.